services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -201,40 +201,45 @@ private String putJavaParamsInNgTranslateFormat(String propertyValue) {
 
         return propertyValue;
     }
-    
+
     @GET
     @Path("index.html")
     public Response getIndexHtmlRedirect() {
         return Response.status(302).location(session.getContext().getUri().getRequestUriBuilder().path("../").build()).build();
     }
 
-
     private String[] getReferrer() {
```

Comment:
I re-wrote this method to use an early-return pattern, as the amount of branching/nesting made it hard to understand what was going on.